client/src/app/notifications/Notification.less

```diff
@@ -51,6 +51,16 @@
     color: var(--color-grey-225-10-35);
     margin-left: 36px;
     margin-right: 16px;
+
+    button {
+      border: none;
+      background-color: transparent;
+      margin: 0;
+      padding: 0;
+      color: var(--color-blue-205-100-50);
```

Comment:
Maybe not only related to this PR, but can we introduce color variables for the Notification styles? Like we do in [other places](https://github.com/camunda/camunda-modeler/blob/develop/client/src/shared/ui/overlay/OverlayDropdown.less#L1). We have a good chance to clean things up here 👍

Reply:
It's a quick thing that is easily handled now 👍

Reply:
Thanks a lot 👍